SPECS/jemalloc/jemalloc.spec

```diff
@@ -0,0 +1,298 @@
+%define      debug_package %{nil}
+Summary:        A General Purpose Malloc Implementation
+Name:           jemalloc
+Version:        5.2.1
+Release:        1%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            http://www.canonware.com/jemalloc/
+Source0:        https://github.com/jemalloc/%{name}/releases/download/%{version}/%{name}-%{version}.tar.bz2
+
+%description
+General-purpose scalable concurrent malloc(3) implementation that emphasizes
+fragmentation avoidance and scalable concurrency support
+
+%package devel
+Summary:        Development files for %{name}
+Group:          Development/Libraries
+Requires:       %{name} = %{version}-%{release}
+
+%description devel
+The %{name}-devel package contains libraries and header files for
+developing applications that use %{name}.
+
+%prep
+%setup -q
+
+%build
+%configure
+make %{?_smp_mflags}
+
+%install
+make install DESTDIR=%{buildroot}
+rm %{buildroot}%{_docdir}/%{name}/jemalloc.html
+
+find %{buildroot}%{_libdir}/ -name '*.a' -exec rm -vf {} ';'
+
+%clean
+rm -rf %{buildroot}
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
+%files
+%{_libdir}/libjemalloc.so.*
+%{_bindir}/jemalloc.sh
+%license COPYING
+%doc README VERSION
+%doc doc/jemalloc.html
+
+%files devel
+%{_includedir}/jemalloc
+%{_bindir}/jemalloc-config
+%{_libdir}/pkgconfig/jemalloc.pc
+%{_bindir}/jeprof
+%{_libdir}/libjemalloc.so
+%{_mandir}/man3/jemalloc.3*
+
+%changelog
+* Mon Dec 14 2020 Henry Li <lihl@microsoft.com> - 5.2.1-1
+- Initial CBL-Mariner import from Fedora 31 (license: BSD).
```

Comment:
>(license: BSD) [](start = 44, length = 14)

The "license" here refers to the license under which the spec is released, which is always MIT for Fedora. The "License" tag refers to the sources, which can have an arbitrary license, thus the need for a separate verification.

2 requests here:
1. Please change "license: BSD" to "license: MIT".
2. Please verify the sources license (I can see that there should be some info available inside "COPYING") and add the phrase "License verified" to the changelog to indicate we've done this work.

The same applies for any other spec we're adding.

Reply:
Resolved.